### DISCUSSION TITLE
Not able to download export

### DISCUSSION THREAD

**Discussion by JasonOui1** [outline/outline#4149] [Self Hosting] [answered]
In the Export section, I generate an export for all my collections. I see a link to "All Collections". When I click on the ... and then Download I am redirected to a page that displays:

`{"ok":false,"error":"not_found","status":404,"message":"Resource not found"}`

I am able to upload images and display them in my environment. I have SMTP variables defined but emailing the link doesn't appear to be working either. I see the following details in the docker log:

`{"templateName":"ExportSuccessEmail","props":{"to":"[obfuscated]"","userId":"1a3a392d-07d6-4424-b6f0-4309f0882fcd","id":"fd7deea8-8e70-4acd-ac59-72a9b7c1c431","teamUrl":"https://outline.domain.com","teamId":"5c8a470e-6d43-408a-9cf3-d5229e291ce9"},"label":"worker","level":"info","message":"EmailTask running"}
{"to":"[obfuscated]","userId":"1a3a392d-07d6-4424-b6f0-4309f0882fcd","id":"fd7deea8-8e70-4acd-ac59-72a9b7c1c431","teamUrl":"https://outline.domain.com","teamId":"5c8a470e-6d43-408a-9cf3-d5229e291ce9","label":"email","level":"info","message":"Email ExportSuccessEmail not sent due to beforeSend hook"}`

Any suggestions as to what I should check in the configuration to allow for the exports to download and/or email?

I'm running v.0.66.0

**Comment by vgwidt** [comment, marked as answer]
Export via the interface doesn't work in the latest community release (I think it provides an incorrect download link), but it has since been fixed in the main branch.  I'm guessing the next release will be pushed out next month.  Doing an [export via API](https://www.getoutline.com/developers#tag/Collections/paths/~1collections.export_all/post) should give you the correct download URL, otherwise you could build your own docker image from source in the meantime.

**Comment by JasonOui1** [comment]
Sadly, I believe the API is the culprit of the problem and the API can lead to additional confusion. While I can call collections.export_all, the fileOperations.redirect provides an output error. 

```
<Error>
   <Code>BadRequest</Code>
   <Message>An error occurred when parsing the HTTP request POST at '/outline/uploads/5c8a470e-6d43-408a-9cf3-d5229e291ce9/1e15cc9d-061c-456c-8b9d-3793ba8ed718/Wiki-export.zip'</Message>
   <Resource>/outline/uploads/5c8a470e-6d43-408a-9cf3-d5229e291ce9/1e15cc9d-061c-456c-8b9d-3793ba8ed718/Wiki-export.zip</Resource>
   <RequestId/>
   <HostId>4b55eab8-e4a6-48e8-a27d-92fd5560e7a9</HostId>
</Error>
```
However the error does give you the location in minio. I was able to find the file in the minio server based on the file path given in the error message.

**Reply by tommoor** [reply]
I released https://github.com/outline/outline/releases/tag/v0.66.1 with a fix for this, the docker image is building now and will be available shortly.

**Reply by JasonOui1** [reply]
I pulled down the latest docker image and I am running v0.66.1. I am able to download the exports. Thank you for the quick response on this.
